src/main/java/com/zipcodewilmington/looplabs/IntegerDuplicateDeleter.java

```diff
@@ -1,8 +1,92 @@
 package com.zipcodewilmington.looplabs;
-
+import java.util.Arrays;
 /**
  * Created by leon on 1/29/18.
  * @ATTENTION_TO_STUDENTS You are forbidden from modifying the signature of this class.
  */
 public final class IntegerDuplicateDeleter extends DuplicateDeleter<Integer> {
+
```

Comment:
Remove all this redundancy.
In your the method bodies of your `IntegerDuplicateDeleter` instantiate a `StringDuplicateDeleter stringDD = new StringDuplicateDeleter(this.array)`.
Then, use `stringDD` to invoke the `removeDuplicates` method.